Vagrantfile

```diff
@@ -148,6 +153,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vbguest.auto_update = false
   end
 
+  # setup VM proxy to system proxy and skip proxy for #{BASE_IP_ADDR}.100/255
+  if Vagrant.has_plugin?("vagrant-proxyconf")
```

Comment:
I would replace this line with a check if `HTTP_PROXY` or `HTTPS_PROXY` are not empty, because the plug-in shouldn't be there if it's not needed in the first place.